### PR TITLE
Add missing :group keys to defcustom forms

### PR DESCRIPTION
**Pull request by purcell**

### CODE REVIEW

File: password-mode.el [hunk 1]
```diff
@@ -44,7 +44,8 @@
 
 Regexps must not contain parentheses for grouping, otherwise your
 match wouldn't work.  Shy groups are OK."
-  :type '(repeat (regexp :tag "Password Regex")))
+  :type '(repeat (regexp :tag "Password Regex"))
+  :group 'password-mode)
```
Comment (alphapapa) [inline password-mode.el]: Forgive the intrusion, just curious: is this required?  The elisp manual says, `If a defcustom does not specify any :group, the last group defined with defgroup in the same file will be used. This way, most defcustom do not need an explicit :group.`

**Comment by alphapapa** [comment]
Forgive the intrusion, just curious: is this required?  The elisp manual says, `If a defcustom does not specify any :group, the last group defined with defgroup in the same file will be used. This way, most defcustom do not need an explicit :group.`

**Comment by purcell** [comment]
Oh, nice: I didn't know that. I guess at the very least it's best to be consistent about it, i.e. either provide or omit `:group` everywhere.

**Comment by juergenhoetzel** [comment]
Thanks a lot! :+1:

**Pull request merged**